### PR TITLE
Database connection

### LINKED ISSUE #13: Create docker database
Create a docker container that runs a database with which the jdbc docker can communicate to the main backend. This database should still be customizable to some extend.

### PR DESCRIPTION
**Pull request by Moltensnor**
- Added the database dump to the project.
- Changed the compose.yaml so it now includes the database and imports the structure
- Added example functions on the account and product frontend that gets data from the database with help from the eventbus

Closes #13

### CODE REVIEW

File: src/main/kotlin/com/ex_dock/ex_dock/database/account/AccountJdbcVerticle.kt [hunk 1]
```diff
@@ -1,47 +1,55 @@
...
-        obj("importance_levels" to rows.iterator().next().getInteger("importance_levels")).encode()
-      })
-    } else {
-      ctx.response().setStatusCode(404).end()
-    }
-  }
-
   private fun getAccountData() {
-    val consumer = eventBus.localConsumer<String>("process.account.getData")
+    val consumer = eventBus.localConsumer<Any>("process.account.getData")
```
Comment (rensPols) [inline src/main/kotlin/com/ex_dock/ex_dock/database/account/AccountJdbcVerticle.kt:26]: The variable consumer isn't very clear about what it is doing. a name like "consumerEventBus" would be clearer about it's purpose.

File: src/main/kotlin/com/ex_dock/ex_dock/database/product/ProductJdbcVerticle.kt [hunk 2]
```diff
@@ -1,6 +1,55 @@
...
+
+  override fun start() {
+    client = Connection().getConnection(vertx)
+    eventBus = vertx.eventBus()
+
+    getAllProducts()
+  }
+
+  private fun getAllProducts() {
+    val consumer = eventBus.localConsumer<Any>("process.products.getAll")
```
Comment (rensPols) [inline src/main/kotlin/com/ex_dock/ex_dock/database/product/ProductJdbcVerticle.kt:24]: This "consumer" variable name isn't relevant. It shoud be something like "productEventBus"

**Review by rensPols — CHANGES_REQUESTED** [review]
Variable names need to be changed. All the functionality looks good!

File: src/main/kotlin/com/ex_dock/ex_dock/database/account/AccountJdbcVerticle.kt (hunk 1 above)
Comment (Moltensnor) [reply src/main/kotlin/com/ex_dock/ex_dock/database/account/AccountJdbcVerticle.kt:26]: I changed the type to JsonObject. The Any was just for testing purposes, but I forget to change them.

File: src/main/kotlin/com/ex_dock/ex_dock/database/product/ProductJdbcVerticle.kt (hunk 2 above)
Comment (Moltensnor) [reply src/main/kotlin/com/ex_dock/ex_dock/database/product/ProductJdbcVerticle.kt]: I changed the variable names so it better says what the consumer is for.

**Review by rensPols — APPROVED** [review]

**Pull request merged**